builtin/provisioners/chef/linux_provisioner.go

```diff
@@ -89,10 +106,10 @@ func (p *provisioner) linuxCreateConfigFiles(o terraform.UIOutput, comm communic
 			if err := p.runCommand(o, comm, "chmod 755 "+hintsDir); err != nil {
 				return err
 			}
-			if err := p.runCommand(o, comm, fmt.Sprintf(chmod, hintsDir, 600)); err != nil {
+			if err := p.runCommand(o, comm, fmt.Sprintf(p.getChmodCommand(), hintsDir, 600)); err != nil {
 				return err
 			}
-			if err := p.runCommand(o, comm, "chown -R root:root "+hintsDir); err != nil {
+			if err := p.runCommand(o, comm, "chown -R root:"+p.getRootGroup()+" "+hintsDir); err != nil {
```

Comment:
I think it would be cleaner if we also made a `chownAix` and `chownLinux` const, just like we did for `chmodXxx`. What do you think?

Reply:
Yeah I can get behind that. It'll help make this change more visible by placing it with its related counterpart, and it'll make the `RootGroup` parameter unnecessary. I'll fix this.